kbase/narrative
# DATAUP-360 reports table

# Description of PR purpose/changes

Adds the table of reports for a completed bulk import cell.
This modifies the results tab to fetch all of the data it needs, including a list of created objects and reports, and feeds that data to its sub-widgets, the created objects viewer and reports viewer.
This will also include a refactor (largely from linting / prettier) of the `kbaseReportView` widget.

(in draft as report view tests are still pending)

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-360

# Testing Instructions
* Details for how to test the PR:
There's dummy data available. Toggle to the Result tab and you should see something like the first image. Toggling the report label should show the second image.
![Screen Shot 2021-02-02 at 12 43 10 PM](https://user-images.githubusercontent.com/2152243/106640671-2ef1b380-653b-11eb-8ff1-f75adc6f3e34.png)
![Screen Shot 2021-02-02 at 12 43 21 PM](https://user-images.githubusercontent.com/2152243/106640682-31540d80-653b-11eb-9051-bbcaf5c0a95f.png)


# Dev Checklist:
